### PR TITLE
Add error messages to login panel 119

### LINKED ISSUE #119: Add error messages to login panel when login fails

### PR DESCRIPTION
**Pull request by garciajordy**
# Login Page

**Devise:**
- Generate all view files.
- Delete all files except sessions/new file.
- Convert HTML to slim.

**Devise/new.slim:**
- Add flash alert messages to form, as devise use flash messages for login.

**i18n:**
- Add missing keys for the login page.

## Rspec

**log_in_spec.rb:**
- Add test to see if the error message is displaying.

_Thanks for reviewing my PR!_ 😄 

fix #119

### CODE REVIEW

File: backend/.ruby-version [hunk 1]
```diff
@@ -1 +1 @@
-3.0.1
+3.0.0
```
Comment (edimossilva) [inline backend/.ruby-version:1]: remove

File: backend/Gemfile [hunk 2]
```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby '3.0.1'
+ruby '3.0.0'
```
Comment (edimossilva) [inline backend/Gemfile:4]: remove

File: backend/Gemfile.lock [hunk 3]
```diff
@@ -433,7 +433,7 @@ DEPENDENCIES
   webpacker (~> 5.0)
 
 RUBY VERSION
-   ruby 3.0.1p64
+   ruby 3.0.0p0
```
Comment (edimossilva) [inline backend/Gemfile.lock:436]: remove

File: backend/app/views/devise/sessions/new.slim [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    = f.label :password
+    = f.password_field :password, autocomplete: 'current-password', class: 'form-control'
+  - if devise_mapping.rememberable?
+    .form-group.form-check
+      = f.check_box :remember_me, class: 'form-check-input'
+      = f.label :remember_me, class: 'form-check-label' do
+        = resource.class.human_attribute_name('remember_me')
+  .form-group
+    = f.submit 'Log in', class: 'btn btn-primary'
+= render 'devise/shared/links'
```
Comment (edimossilva) [inline backend/app/views/devise/sessions/new.slim:21]: add empty line at the bottom of the file

**Review by edimossilva — CHANGES_REQUESTED** [review]

File: backend/.ruby-version (hunk 1 above)
Comment (garciajordy) [reply backend/.ruby-version]: Done!

File: backend/Gemfile (hunk 2 above)
Comment (garciajordy) [reply backend/Gemfile]: Done!

File: backend/Gemfile.lock (hunk 3 above)
Comment (garciajordy) [reply backend/Gemfile.lock]: Done!

File: backend/app/views/devise/sessions/new.slim (hunk 4 above)
Comment (garciajordy) [reply backend/app/views/devise/sessions/new.slim]: Done!

**Review by edimossilva — APPROVED** [review]
Nice!!!!

**Pull request merged**